Front-End/public/styles.css

```diff
@@ -1,29 +1,204 @@
 body {
-    background-color: grey;
-    color: black;
-    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial;
+    margin: 0;
+    width: 100%;
+    font-family: "Roboto", sans-serif;
+    font-weight: 400;
+    font-style: normal;
+}
+
+
+/*navigation bar*/
+.navbar {
+    background-color: #ffffff;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0.5% 5% 0.5% 5%;
+}
+
+.navbar .logo a {
+    color: #000;
+    text-decoration: none;
+    font-size: 1.4rem;
+    font-weight: 700;
+}
+
+.nav-links {
+    display: flex;
+    list-style: none;
+}
+
+.nav-links a {
+    color: #000;
+    margin: 0 0.6rem;
+    text-decoration: none;
+}
+
+
+/*table*/
+.container {
+    background-color: #e1e1e1;
+    padding: 6vh 10vw;
+}
+
+.table {
+    box-shadow: 0 4px 8px 0 #83838333, 0 6px 20px 0 #83838333;
+    background-color: #ffffff;
+    border-radius: 15px;
+    padding: 1.5rem;
+}
+
+.table-title {
+    display: flex;
+    justify-content: space-between;
+    padding-bottom: 15px;
+}
+
+.table-title h2 {
+    font-size: 1.1rem;
+    font-weight: 700;
+    margin: 0;
+}
+
+.table-title a {
+    color: #00c6ff;
     text-align: center;
+    text-decoration: none;
 }
 
-h1 {
-    font-size: 2em;
+.table-container{
+    width: 100%;
 }
 
-.left-aligned {
-    float: left;
+.table-container thead tr th:last-of-type{
+    width: 13%;
+}
 
+.table-container thead tr th:nth-last-of-type(2){
+    width: 20%;
 }
 
-.red{
-    color: red;
+.table-container tr td:first-of-type{
+    width:30%;
 }
-.butonn{
-    background-color: green;
-    border:1px solid orange;
-    padding: 10px
+
+.table-container th {
+    color: #616161;
+    font-weight: 400;
+    font-size: 0.8rem;
+    text-align: left;
+    padding: 5px;
+    user-select: none;
+}
+
+.table-container td {
+    cursor: pointer;
+    border-radius: 10px;
+    padding: 5px;
+}
+
+.esp-profile {
+    display: flex;
+    align-items: center;
 }
-.butonn-destroy{
-    background-color: red;
-    border:10px solid black;
-    padding: 10px
+
+.esp-profile img{
+    width: 1.7rem;
+    margin-right: 5px;
+}
+
+
+/*footer*/
+.footer {
+    display: flex;
+    background-color: #ffffff;
+    padding: 2.5% 5%;
+}
+
+.brand .logo {
+    display: flex;
+}
+
+.brand .logo img {
+    margin: 5px;
+    width: 2.7rem;
+}
+
+.footer {
```

Comment:
The footer should have a fixed position - it should always be at the bottom of the page. The way you've coded it, it's dependant on the body size. Fix that. Either extend the body to the height of the page so the footer stays at the bottom or somehow use absolute positioning. The footer has to not look out of place. If you are not able to replicate the issue on your end, zoom out the web page.